### DISCUSSION TITLE
访问主页不需要账号密码吗，还是可以设置

### DISCUSSION THREAD

**Discussion by lsjcx** [AlistGo/alist#2221] [General]
访问主页不需要账号密码吗，还是可以设置，
现在挂载之后，访问主页不需要验证，直接可以看到，挂在的网盘文件。挂在服务器上的话，只要IP暴露的话，不是很容易被别人看到吗？

**Comment by zch11230** [comment]
添加个元信息，路径为/，设置密码并应用到子文件夹，下面根据需要设置是否写入。这样一打开主页就会提示输密码了。

**Reply by panziqiang007** [reply]
这个方法不错
